src/modules/keyboardmanager/README.md

```diff
@@ -0,0 +1,96 @@
+# Keyboard Manager 
+
+The Keyboard Manager (KBM) is a keyboard remapper that allows a user to
+redefine keys on their keyboard (ex. swapping the letter A and D) as
+well as shortcuts (Ctrl+C to Ctrl+A). Users can use these remappings as
+long as KBM is enabled and PowerToys is running in the background.
+
+![alt text][example]
+
+# 1\. Get Started  
+
+## General Settings
+To create mappings with Keyboard Manager, you have the option of launching either the Remap Keyboard UI by clicking the \<Remap a Key\> button or Remap Shortcuts UI by clicking the \<Redefine a shortcut\> button.
+
+
+## Remap Key to Key
+To remap a key to another key, click the \<Remap a Key\> button to
+launch the Remap Keyboard UI. When first launched, you are met
+with no predefined mappings and must click the \<+\> button to add
+a new remap. From there, the user selects the key whose output
+they want to ***change*** as the “Original Key” and then keys new
+output as the “New Key”. For example, if you want to press A and
+get B, Key A would be your “Original Key” and Key B would be your
+“New Key”. If you want to swap keys, add another remapping with
+B as your Original and Key A as your New.
+
+![alt text][remapkey]
+
+## Remap Shortcuts (OS-only)
+Currently you are only able to remap OS-level shortcuts,
+app-specific shortcuts are coming soon\! To change how you invoke
+a particular shortcut, click the \<Remap Shortcut\> button to
+launch the Edit Shortcuts UI. When first launched, you are met
+with no predefined mappings and must click the \<+\> button to add
+a new remap. The Original shortcut is the shortcut you want to
+change and the New Shortcut is the shortcut you want to change it
+to. Ex. If you want Ctrl+C to paste, Ctrl+C is the Original
+Shortcut and Ctrl+V is the New Shortcut. There are a few rules to shortcuts:
+   
+- Shortcuts must begin with a modifier key (Ctrl, Shift, Alt, and Winkey)
+- Shortcuts must end with an action key (all non-modifier keys and non Fn keys)\* 
+- Shortcuts cannot be longer than 3 keys  
+
+![alt text][remapshort]
+
+### System Reserved shortcuts 
+- Winkey + L (Locking your computer) and Ctrl + Alt + Del cannot be remapped (as Original Shortcut) as they are reserved by the Windows OS.
```

Comment:
Neither original nor new shortcut. We have warnings for both since even if the user remaps something like Ctrl+A to Win+L it will not lock the screen.